src/frequenz/client/dispatch/_cli_types.py

```diff
@@ -5,13 +5,23 @@
 
 import json
 from datetime import datetime, timedelta, timezone
+from sre_constants import IN
```

Comment:
I bet this was Tab-oriented-programming + auto-import :laughing: 
```suggestion
```

Reply:
it was more desperation, but this is a draft after all :P